app/view_objects/apply_invalid_data_view.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class ApplyInvalidDataView
+  include ActionView::Helpers::UrlHelper
+  include ActionView::Helpers::TagHelper
+
+  def initialize(apply_application)
+    @apply_application = apply_application
+    @invalid_fields = populate_invalid_fields
+  end
+
+  def summary_content
+    return pluralised_invalid_answers_summary if invalid_fields.size > 1
+
+    I18n.t("views.apply_invalid_data_view.invalid_answer_summary")
```

Comment:
```suggestion
    I18n.t("views.apply_invalid_data_view.invalid_answers_summary", count: invalid_fields.size)
```

Reply:
done in https://github.com/DFE-Digital/register-trainee-teachers/pull/1150/commits/211708543fdff8bc5b397be33f4bbbbf298ad964